packages/popup/src/pages/CurrentSeed/index.js

```diff
@@ -66,12 +66,15 @@ function CurrentSeed() {
   const {setLoading} = useLoading()
 
   useEffect(() => {
-    setAccountNamePlaceholder(`Seed-1-${hdGroup[0]?.account?.length + 1}`)
+    setAccountNamePlaceholder(
+      `${t('seedPrefix')}1-${hdGroup[0]?.account?.length + 1}`,
```

Comment:
create account name with hardcode not i18n

Reply:
done